hw/Makefile

```diff
@@ -115,22 +111,6 @@ $(filter lc_ctrl_reg,$(ips_reg)): lc-state-enc
 lc-state-enc:
 	cd ${PRJ_DIR} && ./util/design/gen-lc-state-enc.py ${LC_BUFFER}
 
-regs-header: $(ips_reg_header)
```

Comment:
Are we sure these targets are not required anymore?
I am not certain...

@a-will @timothytrippel maybe you know?

Reply:
I don't think we do need them. These targets are generated by bazel now. You'll notice they aren't part of the `all` PHONY target, either.

I'm not sure what workflows are currently in use for downstream users, though, if that's important.

Reply:
@timothytrippel Rust top const are stil generated by topgen form this Makefile, so we can't remove it without implementing correct bazel replacement.

Reply:
> I don't think we do need them. These targets are generated by bazel now. You'll notice they aren't part of the `all` PHONY target, either.
> 
> I'm not sure what workflows are currently in use for downstream users, though, if that's important.

Upstream TockOS use Rust register definitions and generated top "header".
Also we have some internal usage in `sw/host`.
It is easy to check what we generated by inspecting '*.tpl' files and python topgen code..